tests/skipped_tests.tbl

```diff
@@ -483,6 +483,15 @@ tests/third_party/cupy/manipulation_tests/test_shape.py::TestRavel::test_ravel
 tests/third_party/cupy/manipulation_tests/test_shape.py::TestReshape::test_reshape_zerosize
 tests/third_party/cupy/manipulation_tests/test_shape.py::TestReshape::test_reshape_zerosize2
 
+tests/third_party/cupy/manipulation_tests/test_kind.py::TestKind::test_asarray_chkfinite
+tests/third_party/cupy/manipulation_tests/test_kind.py::TestKind::test_asarray_chkfinite_non_finite_vals
+tests/third_party/cupy/manipulation_tests/test_kind.py::TestKind::test_require_C_and_F_flags
```

Comment:
it would be better to mute the tests through `@pytest.amrk.skip` with exact reason stated